gcc/config/riscv/t-riscv

```diff
@@ -8,7 +8,8 @@ riscv-builtins.o: $(srcdir)/config/riscv/riscv-builtins.cc $(CONFIG_H) \
   $(DIAGNOSTIC_CORE_H) $(OPTABS_H) $(RISCV_BUILTINS_H) \
   $(srcdir)/config/riscv/riscv-ftypes.def \
   $(srcdir)/config/riscv/riscv-vector-builtins-types.def \
-  $(srcdir)/config/riscv/riscv-modes.def
+  $(srcdir)/config/riscv/riscv-modes.def \
+  $(srcdir)/config/riscv/riscv-builtins-rvp.def
```

Comment:
if you included it in riscv-builtin.cc, no need to add here

Reply:
I think this line is for that if `riscv-modes.def` has been modified, the `make`  tool should be able to detect this modification and rebuild it.